### DISCUSSION TITLE
Transfer learning with MONAI-Label

### DISCUSSION THREAD

**Discussion by rfrs** [Project-MONAI/MONAILabel#1713] [Q&A]
I would like to know if it possible to use MONAI-Label and do transfer learning in the following scenario. 

Let's assume you train one model in dataset A with X labels. Then you have access to dataset B with Y labels (X<Y). Can this work somehow? So far, always gives me back an error message because I have too many classes. 

Best wishes

**Comment by diazandr3s** [comment]
Hi @rfrs,

Great meeting you.
Currently, this sort of transfer learning isn't possible in MONAI Label. A model with different number of segments is considered as a different task. This shouldn't be like this as there is "knowledge" that you could transfer from one task to the other. But that's a limitation of the framework :/
We welcome any contribution to address this limitation.
